### PR TITLE
conditional rendering

### LINKED ISSUE #9: Conditional rendering of nomad_job
## Reason
Another module will reuse the current module. Hive docker image should be available somewhere.

## Description
To be able to reuse the current module we need a conditional rendering of nomad_job. 
This part of the code

```text
# todo: optional rendering START (docker image may be already available in registry)
      artifact {
        // vagrant-hashistack:minio
        source = "s3::http://127.0.0.1:9000/dev/tmp/hive_local.tar"
        options {
          aws_access_key_id     = "minioadmin"
          aws_access_key_secret = "minioadmin"
        }
      }

      config {
        load = "hive_local.tar"
        image = "fredrikhgrelland/hive:local"
        command = "hivemetastore"
      }
# todo: optional rendering END
```

ref: https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/master/conf/nomad/hive.hcl#L86-L101 

## Solutions
- [ ] fast&dirty hot-fix with nomad_job fix #10 
- [x] proper conditional rendering of nomad job
- [ ] ...

### PR DESCRIPTION
**Pull request by zhenik**
## Closes/fixes/resolves issue(s)?
- Closes #9 
## What was added/changed/fixed?
- [added] conditional rendering of image
- [remove] 2nd redundant job
- [update] docs, example
## Checklist (after created PR)
- [X] Added reviewers
- [X] Added assignee(s)
- [X] Added label(s)
- [X] Added to project
- [X] Added to milestone

### CODE REVIEW

**Review by claesgill — DISMISSED** [review]
LGTM

**Review by dangernil — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ module "hive" {
 ```
 
 ### Verifying setup
+
```
Comment (claesgill) [inline README.md:148]:
```suggestion
```

Comment (zhenik) [reply README.md:148]: yeah. I had to trigger new actions run, due I cannot cancel this run https://github.com/fredrikhgrelland/terraform-nomad-hive/actions/runs/321493269 . I tried to cancel, but it did not work

Comment (claesgill) [reply README.md:148]: Oh :sweat_smile:

**Review by dangernil — APPROVED** [review]

**Review by claesgill — APPROVED** [review]
:vulcan_salute:

**Pull request merged**